must-gather/collection-scripts/gather_operator

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+OSC_PATH=${BASE_COLLECTION_PATH}/sandboxed-containers/operator
+mkdir -p ${OSC_PATH}
+
+/usr/bin/oc describe csv -n openshift-sandboxed-containers-operator > ${OSC_PATH}/csv
+/usr/bin/oc describe subscription -n openshift-sandboxed-containers-operator > ${OSC_PATH}/subscription
+
+deploymentUnits=(pods deployments statefulsets deploymentconfigs)
+
+for deploymentUnit in "${deploymentUnits[@]}"; do
+	echo "$deploymentUnit using kata runtime class still running:"
+
+	/usr/bin/oc get $deploymentUnit -A -o json | jq -r '.items[] | select(.spec.runtimeClassName | test("kata")).metadata.name' 2>/dev/null > ${OSC_PATH}/${deploymentUnit}
+
+done
```

Comment:
could this whole thing be under a `deployments` directory, under https://github.com/openshift/sandboxed-containers-operator/blob/0c4e7aaab1df9ae92b66b28ceb1528eab9c7bd23/must-gather/collection-scripts/gather_sandboxed_containers_logs#L23 ?